sdk/eventhub/event-processor-host/package.json

```diff
@@ -73,7 +73,7 @@
     "debug": "^4.1.1",
     "path-browserify": "^1.0.0",
     "tslib": "^1.10.0",
-    "uuid": "^3.3.2"
+    "uuid": "^8.1.0"
```

Comment:
@chradek, @ramya-rao-a: Do you think `uuid` should be updated in `event-processor-host`?  If we want to keep it consistent with other packages then yes it should be updated, but I know `event-processor-host` is already an exceptional package since it uses older versions of other packages (like `@azure/event-hubs@^2.1.4`).

Reply:
I think it's fair to update uuid here. We may not do a release right away but I don't really see a reason not to update it. (Presuming we also update it in event-hubs v2 if we need to release it again in the future)